docs/page.css

```diff
@@ -61,6 +61,7 @@ body {
 	padding-bottom: var(--page-padding);
 	padding-right: var(--page-padding);
 	padding-left: calc(var(--page-padding) + var(--panel-width));
+	word-break: break-word;
```

Comment:
Do we really need this new CSS in `body`? It will affect the entire documentation...

Reply:
If not, we need it in two places, P for links and CODE for long lines.

Reply:
Um, even when I disable the CSS in the inspector the new meta code looks still good.

Reply:
@Mugen87 list of bad examples in safari:

- introduction/Compatibility check
- introduction/Faq
- How to use post-processing
- Testing with npm
- Text geometry
- Text buffer geometry
- MeshStandard materials

Reply:
![](https://habrastorage.org/webt/l_/_w/mf/l__wmfpbamvtnmtxsab-npfrm38.jpeg)

Reply:
Okay, that makes it more clear 👍